packages/expo-cli/src/commands/export/mergeAppDistributions.ts

```diff
@@ -7,7 +7,7 @@ import { Project, XDLError } from 'xdl';
 
 import Log from '../../log';
 
-type SelfHostedIndex = ExpoAppManifest & {
+type SelfHostedIndex = ExpoAppManifestWithSdk & {
```

Comment:
Basing the sdk requirement off of this function: https://github.com/expo/expo-cli/pull/3547/files#diff-1a0c6311e2ebf725a99acb8988c676a4f260ca080d36bf4eee0dc8b617a0125fR15